apps/server/src/modules/lesson/service/lesson.service.ts

```diff
@@ -47,6 +52,8 @@ export class LessonService implements AuthorizationLoaderService {
 
 		await this.lessonRepo.save(updatedLessons);
 
+		this.logger.log({ action: 'Deleted User Data From Lesson for user ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of updated lesson without userId. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114